### ISSUE TITLE
Unexpected TypeError while running tomlkit.loads()

### ISSUE THREAD

**Issue by oakkitten** [sdispater/tomlkit#63]
Run `tomlkit.loads("foo bar")`, observe that it raises, as expected, 

    UnexpectedCharError: Unexpected character: 'b' at line 1 col 4

Run `tomlkit.loads("hello there")`, observe that it suddenly raises

    TypeError: __init__() missing 1 required positional argument: 'char'

**Comment by MrGreenTea** [comment]
This looks like it's trying to parse `there` as a bool. The same happens for all pairs where the value starts with `'t'`

```python
tomlkit.loads("t t")
tomlkit.loads("hello=t")
tomlkit.loads("hello=there")
tomlkit.loads("key tru")
```

**Comment by MrGreenTea** [comment]
This also happens when trying to parse `'f'`

```python
tomlkit.loads("f f")
tomlkit.loads("hello=f")
tomlkit.loads("hello=f")
tomlkit.loads("key fal")
```

This happens in `Parser._parse_value` because it only checks if the char is the begining of a bool type with
 ```python
c == BoolType.FALSE.value[0]
```

If parsing fails here it should most likely raise some `ParseError`

**Comment by frostming** [comment]
I've created a fork of tomlkit v0.7.0 with the issue fixed, try it:

GitHub: https://github.com/frostming/atoml
PyPI: https://pypi.org/project/atoml

**Comment by frostming** [comment]
Fixed by 3c3f469
